sqlmesh/schedulers/airflow/dag_generator.py

```diff
@@ -74,6 +74,8 @@ def __init__(
             t.Callable[[t.Dict[str, t.Any]], BaseSensorOperator]
         ],
         sensor_mode: str,
+        high_water_mark_sensor_args: t.Optional[t.Dict[str, t.Any]],
+        high_water_mark_external_sensor_args: t.Optional[t.Dict[str, t.Any]],
```

Comment:
Calling external sensors "high watermark" was a mistake and likely a result of a copy paste. Can we just all this `external_sensor_args`?

Reply:
https://github.com/TobikoData/sqlmesh/pull/2850/commits/66e41db19c895065b66bfe41d4e99c1621faeb19